docs/blog/newsletter-february-2020.rst

```diff
@@ -0,0 +1,96 @@
+* Audit your docs, archiving or updating content that people aren't reading. Alongside PMs, determine which content is outdated or unnecessary, and then divide them among writers on the team. And if you're struggling to persuade colleagues that an audit is necessary, our next article might just help.
+
+--------------------------
+The costs of outdated docs
+--------------------------
+
+This month's discussions included a brief but fruitful conversation about the tangible costs of outdated, inaccurate, or just plain poorly written documentation. It can be helpful to keep data about the value of good documentation on hand, and our documentarians turned up several reports and studies to consider:
+
+- `The State of Unassisted Support 2014 <http://cdn.swcdn.net/creative/pdf/Whitepapers/SW-WP-The-State-of-Unassisted-Support-2014.pdf>`_, Technology Services Industry Association (TSIA). Includes a graph that compares the cost of self-service support with the costs of other methods.
+- `A Good User's Guide Means Fewer Support Calls and Lower Support Costs <www.jstor.org/stable/43095065>`_ by Cathy J. Spencer and Diana Kilbourn Yates. GE Information Services saved $1 million in support costs per year by providing good user documentation with software products.
+- `Currents, A Seasonal Report on Developer Trends in the Cloud: Open Source Edition <https://www.digitalocean.com/currents/october-2018>`_, Digital Ocean. For open source technologies, great documentation can reduce the barrier to entry for contributors, and it's something companies are looking for when they evaluate open source products.
+
+----------------------------------------------
+Dealing with finding inaccuracies in your docs
+----------------------------------------------
+
+Although we strive to write accurate documentation, inevitably there are times when errors are made that require addressing. This month our community talked about what to do when you realise there are inaccuracies in published articles.
+
+The question of blame is unavoidable. Should writers have their work scrutinized for small errors? Many felt problems in documentation should rather be attributed to process failure. If you have to complete a document in a limited time, the pressure of the crunch can result in not enough time to check, and thereby faulty docs. Others didn't want to frame it as a failure: just a chance to improve both the doc and processes. But even if you have a good pre-publication process, errors may arise from other factors: for example, testing the docs but within the wrong environment, meaning they don't match the end user experience.
+
+What most agreed on is that the documentation should be revised and corrected as soon as any errors are discovered. If errors are a big problem, teams can establish a process of reviews and approvals to check for accuracy. Another option suggested was be to have a QA function test the documentation.
```

Comment:
rm "be" and trim last sentence: "Another suggestion was to have QA test the documentation."

Reply:
QA function?

Reply:
ty, fixed